### ISSUE TITLE
Creating Directory instead of Converting from Semafor to Semeval 

### ISSUE THREAD

**Issue by yongzx** [akb89/pyfn#18]
When I run the `pyfn convert` command to convert from SEMAFOR CoNLL format into Semeval XML, I run into `IsADirectoryError: [Errno 21] Is a directory` error. 

I found the command now creates the directory with the inclusion of the filename `/home/zxy485/zxy485gallinahome/week1/pyfn/experiments/xp_001/output/test.predicted.xml/`

```
$ pyfn convert \
>   --from semafor \
>   --to semeval \
>   --source /home/zxy485/zxy485gallinahome/week1/pyfn/experiments/xp_001/data/test.frame.elements \
>   --target /home/zxy485/zxy485gallinahome/week1/pyfn/experiments/xp_001/output/test.predicted.xml \
>   --sent /home/zxy485/zxy485gallinahome/week1/pyfn/experiments/xp_001/data/test.sentences
INFO - Marshalling pyfn.AnnotationSet objects to SEMEVAL XML...
INFO - Marshalling pyfn.AnnotationSet objects to SEMEVAL XML...
INFO - Marshalling pyfn.AnnotationSet objects to SEMEVAL XML...
INFO - Saving output to /home/zxy485/zxy485gallinahome/week1/pyfn/experiments/xp_001/output/test.predicted.xml
INFO - Saving output to /home/zxy485/zxy485gallinahome/week1/pyfn/experiments/xp_001/output/test.predicted.xml
INFO - Saving output to /home/zxy485/zxy485gallinahome/week1/pyfn/experiments/xp_001/output/test.predicted.xml
Traceback (most recent call last):
  File "/home/zxy485/.local/bin/pyfn", line 10, in <module>
    sys.exit(main())
  File "/home/zxy485/.local/lib/python3.6/site-packages/pyfn/main.py", line 198, in main
    args.func(args)
  File "/home/zxy485/.local/lib/python3.6/site-packages/pyfn/main.py", line 91, in _convert
    args.excluded_annosets)
  File "/home/zxy485/.local/lib/python3.6/site-packages/pyfn/marshalling/marshallers/semeval.py", line 128, in marshall_annosets
    excluded_sentences, excluded_annosets)
  File "/home/zxy485/.local/lib/python3.6/site-packages/pyfn/marshalling/marshallers/semeval.py", line 110, in _marshall_annosets
    pretty_print=True)
  File "src/lxml/etree.pyx", line 2048, in lxml.etree._ElementTree.write
  File "src/lxml/serializer.pxi", line 721, in lxml.etree._tofilelike
  File "src/lxml/serializer.pxi", line 780, in lxml.etree._create_output_buffer
  File "src/lxml/serializer.pxi", line 770, in lxml.etree._create_output_buffer
IsADirectoryError: [Errno 21] Is a directory
```

**Comment by akb89** [comment]
Ok I'm on it. Currently re-checking the whole pipeline with training to make sure it's functional.

**Comment by akb89** [comment]
Solved in v1.3.6. Let me know if it works for you.

**Comment by yongzx** [comment]
Thank you!
I am retraining SEMAFOR now. I will let you know whether it works after the training!

**Comment by yongzx** [comment]
Unfortunately I am still running into the error.
```
$ pip show pyfn
Name: pyfn
Version: 1.3.6
Summary: A python module to process data for Frame Semantic Parsing
Home-page: https://gitlab.com/akb89/pyfn
Author: Alexandre Kabbach
Author-email: akb@3azouz.net
License: MIT
Location: /home/zxy485/.local/lib/python3.6/site-packages
Requires: mmh3, pyyaml, pytz, lxml
Required-by:
$ $ pyfn convert \
>   --from semafor \
>   --to semeval \
>   --source /home/zxy485/zxy485gallinahome/week1-4/pyfn/experiments/xp_001/data/test.frame.elements \
>   --target /home/zxy485/zxy485gallinahome/week1-4/pyfn/experiments/xp_001/output/test.predicted.xml \
>   --sent /home/zxy485/zxy485gallinahome/week1-4/pyfn/experiments/xp_001/data/test.sentences
INFO - Marshalling pyfn.AnnotationSet objects to SEMEVAL XML...
INFO - Marshalling pyfn.AnnotationSet objects to SEMEVAL XML...
INFO - Marshalling pyfn.AnnotationSet objects to SEMEVAL XML...
INFO - Saving output to /home/zxy485/zxy485gallinahome/week1-4/pyfn/experiments/xp_001/output/test.predicted.xml
INFO - Saving output to /home/zxy485/zxy485gallinahome/week1-4/pyfn/experiments/xp_001/output/test.predicted.xml
INFO - Saving output to /home/zxy485/zxy485gallinahome/week1-4/pyfn/experiments/xp_001/output/test.predicted.xml
Traceback (most recent call last):
  File "/home/zxy485/.local/bin/pyfn", line 10, in <module>
    sys.exit(main())
  File "/home/zxy485/.local/lib/python3.6/site-packages/pyfn/main.py", line 201, in main
    args.func(args)
  File "/home/zxy485/.local/lib/python3.6/site-packages/pyfn/main.py", line 91, in _convert
    args.excluded_annosets)
  File "/home/zxy485/.local/lib/python3.6/site-packages/pyfn/marshalling/marshallers/semeval.py", line 128, in marshall_annosets
    excluded_sentences, excluded_annosets)
  File "/home/zxy485/.local/lib/python3.6/site-packages/pyfn/marshalling/marshallers/semeval.py", line 110, in _marshall_annosets
    pretty_print=True)
  File "src/lxml/etree.pyx", line 2048, in lxml.etree._ElementTree.write
  File "src/lxml/serializer.pxi", line 721, in lxml.etree._tofilelike
  File "src/lxml/serializer.pxi", line 780, in lxml.etree._create_output_buffer
  File "src/lxml/serializer.pxi", line 770, in lxml.etree._create_output_buffer
IsADirectoryError: [Errno 21] Is a directory
```

**Comment by akb89** [comment]
Weird, I cannot replicate the error. Let's try the followings:
- I've released v1.3.7 since, can you try with this one? 
- what happens when you run the script `./score.sh -x 001 -p semafor -s test -f gold`? 
- is it possible that there was a problem during the generation of the `test.frame.elements` file during decoding? Could you try and re-run `./semafor.sh -m decode -x 001 -s test` and see what happens?
